### PR TITLE
Add workflow for validating planets.json

### PR DESCRIPTION
**Pull request by sky1045**
After validation of this validation(?), uploading to S3, Invalidation of cloudfront will be added

### CODE REVIEW

File: planets/schema.json
Comment (moreal) [inline planets/schema.json:1]: There is already a JSON Schema of planet registry. You can reuse it.

https://github.com/planetarium/json-schema/blob/cc3f8f11d36a90f3649406cb4dcd10c2d2eb4ca9/NineChronicles/2023-10/PlanetRegistry.schema.json

Comment (sky1045) [reply planets/schema.json:1]: woah is there any usecase of it?? thank you for letting me know

Comment (moreal) [reply planets/schema.json:1]: Maybe there is no usecases yet but @longfin has made it when introducing `planets` system 👀

Comment (sky1045) [reply planets/schema.json:1]: as i read it, the bridge config isnt included in the lates schema, so we need to update it first

Comment (eseiker) [reply planets/schema.json:1]: Opened a PR to amend the schema https://github.com/planetarium/json-schema/pull/3

**Review by eseiker — APPROVED** [review]

**Pull request merged**